src/components/ebay-checkbox/test/test.server.js

```diff
@@ -0,0 +1,36 @@
+const expect = require('chai').expect;
+const testUtils = require('../../../common/test-utils/server');
+const rootTag = `span.checkbox`;
+const inputTag = `input.checkbox__control`;
+const iconTag = `span.checkbox__icon`;
+
+test('renders default checkbox', context => {
+    const $ = testUtils.getCheerio(context.render());
+    expect($(rootTag).length).to.equal(1);
+    expect($(inputTag).length).to.equal(1);
+    expect($(iconTag).length).to.equal(1);
+});
+
+test('renders checkbox with value and name', context => {
```

Comment:
I don't think this test adds anything beyond the 'handles pass-through html attributes' test below.